### ISSUE TITLE
clean up '<g type="ekthesis"/>' in XML

### ISSUE THREAD

**Issue by paregorios** [DCLP/dclpxsltbox#300]
A number of XML files have unexpected encoding like:

```xml
<lb n="3"/><g type="ekthesis"/>
```

This is, to my understanding, wrong. What we want (per the long discussion archived on issue ) is xml like:

```xml
<lb n="3" rend="outdent"/>
```

Unless there's something I'm not understanding ( @rla2118 @jcowey @HolgerEssler ? ), the XML files with the ```<g type="ekthesis"/>``` need to be converted.

**Comment by paregorios** [comment]
This is a blocker for #295

**Comment by jcowey** [comment]
Yes the files in which `<g type="ekthesis"/>` appear should please be changed so that they then contain `<g type="outdent"/>`

**Comment by paregorios** [comment]
@jcowey I think you mean ```<lb type="outdent"/>```, yes?

**Comment by jcowey** [comment]
Yes, of course, I did. Sorry, slapped in a comment without thinking too much. :).

**Comment by paregorios** [comment]
This is more complicated than I realized. There are cases where users have entered ```<g type="ekthesis"/>``` somewhere other than at the beginning of the line and sometimes with intervening whitespace. For example:

 - TM69033: ```<lb n="1" xml:id="div1-lb1"/><g type="rho-cross"/> <g type="ekthesis"/>```
 - TM63877: ```<lb n="5"/><supplied reason="lost"><g type="ekthesis"/></supplied>```
 - TM63877 : ```<lb n="11"/><gap reason="lost" extent="unknown" unit="character"/> <g type="ekthesis"/>κ```

The classes of such use (and there are probably more) need to be defined, and decisions taken about how to deal with each of them.

As we are about out of time before final release, I am punting this task to the post-release milestone.
